inference-engine/src/mkldnn_plugin/nodes/mkldnn_strided_slice_node.cpp

```diff
@@ -157,6 +187,23 @@ void MKLDNNStridedSliceNode::getSupportedDescriptors() {
         fillingInParameters(attrs.end, END_ID, attrs.endDims[0], 0);
     if (attrs.strideDims.size())
         fillingInParameters(attrs.stride, STRIDE_ID, attrs.strideDims[0], 1);
+    if (attrs.axesDims.size()) {
+        fillingInParameters(attrs.axes, AXES_ID, attrs.axesDims[0], 0);
+        std::vector<int> beginTmp(outputRank, 0);
+        std::vector<int> endTmp(outputRank, -1);
+        std::vector<int> strideTmp(outputRank, 1);
+        size_t i = 0lu;
+        for (auto a : attrs.axes) {
```

Comment:
Spec says that axes can be negative. Should to normalize it before using
Also add tests with negative axes please

Reply:
Done.